V2.16-RUN-14155-Expose-GPU-health (#612)
run-ai/docs: docs/admin/admin-ui-setup/dashboard-analysis.md

The following analysis can apply to both GPU and CPU dashboards.

### Total and Ready GPU or CPU Nodes

The *Indicators* panel of the *GPU Overview Dashboard* displays the total number of GPU nodes, the number of ready GPU nodes, the total number of GPUs, and the total number of ready GPUs.

The *Indicators* panel of the *CPU Overview Dashboard* displays the total number of CPU nodes, the number of ready CPU nodes, the total number of CPUs, and the total number of ready CPUs.

These panes help calculate the number of available (unscheduled) resources in the platform.

> * **Total GPU/CPU Nodes**&mdash;indicates the number of nodes configured to work with the platform.

Reviewer: I would use "sum total of nodes in all clusters connected to the platform" instead of "number of nodes configured to work with the platform

> * **Total GPU/CPU Nodes**&mdash;indicates the number of nodes configured to work with the platform.

Author: ```suggestion
* **Total GPU/CPU Nodes**&mdash;the total number of GPUs or CPUs in all the clusters that are connected to the platform.
```

> * **Total GPU/CPU Nodes**&mdash;indicates the number of nodes configured to work with the platform.

Author: ```suggestion
* **Total GPU/CPU Nodes**&mdash;indicates the sum total of nodes in all clusters connected to the platform.
```

* **Ready GPU/CPU Nodes**&mdash;indicates the number of nodes that are available to the scheduler. This is calculated by subtracting the number of unscheduled nodes from the total number of nodes.

> * **Total GPUs/CPUs**&mdash;indicates the total number of GPUs or CPUs configured to work with the platform.

Reviewer: The total number of GPUs/CPUs in all clusters connected to the platform

> * **Total GPUs/CPUs**&mdash;indicates the total number of GPUs or CPUs configured to work with the platform.

Author: ```suggestion
* **Total GPUs/CPUs**&mdash;indicates ihe total number of GPUs/CPUs in all the clusters that are connected to the platform.
```

* **Ready GPUs/CPUs**&mdash;indicates the number of GPUs or CPUs that are available to work with the scheduler. This is calculated by subtracting the number of unscheduled GPUs or CPUs from the total number of GPUs or CPUs.

> The *Free GPUs* graph displays the number of nodes and how many GPUs or CPUs are configured in each node.

Reviewer: displays the number of free GPUs/CPUs on each node

> The *Free GPUs* graph displays the number of nodes and how many GPUs or CPUs are configured in each node.

Author: ```suggestion
The *Free GPUs* graph displays the number of free GPUs or CPUs on each node.
```
